docs/en_US/nnicli.md

```diff
@@ -0,0 +1,41 @@
+# NNI Client
+
+NNI client is a python API of `nnictl`, which implements the most common used commands. User can use this API to control their experiments in python code instead of command line. Here is an example:
+
+```
```

Comment:
Might submit this file in a separate PR.

Reply:
Sure. These files were introduced by mistake.